src/index.ts

```diff
@@ -200,3 +259,28 @@ export interface ErrorResultType {
   code: string;
   message: string;
 }
+
+export interface ReadProgramSpaceResultType {
+  programSpaceData: string;
+}
+
+export interface WriteProgramSpaceResultType {
+  isSuccess: boolean;
+}
+
+export enum OperationModes {
```

Comment:
Enums should be singular 
```suggestion
export enum OperationMode {
```